index.bs

```diff
@@ -385,15 +385,15 @@ like
 
 <pre><code class="lang-javascript">
   class ReadableStream {
-    constructor(underlyingSource = {}, { size, highWaterMark } = {})
+    <l>[[#rs-constructor|constructor]]</l>(underlyingSource = {}, { size, highWaterMark } = {})
```

Comment:
Personally I find this more confusing than just doing `<a href="#rs-constructor">constructor</a>`. Does that work, or is only `<l>` allowed inside `<pre>` blocks?

Reply:
Just tested it. Works as well.